packages/grafana-ui/src/utils/storybook/withCenteredStory.tsx

```diff
@@ -1,34 +1,13 @@
 import { DecoratorFn } from '@storybook/react';
-import React from 'react';
 
-interface CenteredStoryProps {
-  children: React.ReactNode;
-  horizontal?: boolean;
-  vertical?: boolean;
-}
-const CenteredStory = ({ horizontal, vertical, children }: CenteredStoryProps) => {
-  return (
-    <div
-      style={{
-        minHeight: '100%',
-        width: '100%',
-        display: 'flex',
-        alignItems: vertical ? 'center' : 'flex-start',
-        justifyContent: horizontal ? 'center' : 'flex-start',
-      }}
-    >
-      {children}
-    </div>
-  );
-};
+/** @deprecated Stories should not be centered on the page. This decorator now does nothing */
+export const withNotCenteredStory: DecoratorFn = (story) => story();
```

Comment:
I assume we manually go through the stories to remove these from them in the future?

Reply:
yup!